samples/nextjs/jss-create.js

```diff
@@ -108,3 +109,14 @@ function setPrerender(prerender) {
       break;
   }
 }
+
+/**
+ * Switch development next.config.js to production config
+ */
+function setNextConfig() {
+  const nextConfig = path.join(__dirname, 'next.config.js');
+  const prodConfig = path.join(__dirname, 'next.config.prod.js');
```

Comment:
Even though this wouldn't end up in the resulting app created via `jss create`, calling this config "production" might cause confusion with the production build (e.g. `npm run start:production`). Can't think of anything better though... `next.config.template.js`??

Reply:
Good call, I renamed it to `next.config.base.js`